snippets/c++/diamondSquare.md

```diff
@@ -0,0 +1,74 @@
+# Diamond Square
+
+## Code
+
+```C++
+int main()
+{
+    int n;
+    cout << "Enter the value of n : ";
```

Comment:
What is 'n' here? The number of rows, columns, or both? Whatever it is, comment out in the code

Reply:
@kaustubhgupta I have made few changes. Please check.
n is the number of rows as mentioned in the problem statement

Reply:
I know n is the number of rows but how will someone who is viewing the code will get it? That's why it was suggested and now you have made the change, it's fine now